### DISCUSSION TITLE
noScribe: new App for transcribing Interviews

### DISCUSSION THREAD

**Discussion by kaixxx** [pyannote/pyannote-audio#1379] [Show and tell]
![noScribe App](https://github.com/ggerganov/whisper.cpp/assets/3353272/4f38c2b3-0b61-481d-b504-3be393714682)
 
**>>> [noScribe on GitHub](https://github.com/kaixxx/noScribe)**
 
The main purpose of this app is to transcribe interviews for qualitative research or journalistic use. It uses **pyannote** and [**whisper.cpp**](https://github.com/ggerganov/whisper.cpp). It also includes a nice **MS Word-interface** to review, verify and correct the resulting transcript. 

The goal was to create an easy-to-use interface for my students and other non techie people (like my colleagues from the humanities department ;) ). Take a look, I appreciate any reactions or suggestions! Also: Please spread the word if you know people that might benefit from this app.

For now, the app is running on windows only (installer included). But I would really appreciate any help in bringing this to mac and linux too. It’s all open source (GPL-3.0), programmed in python and prepared for cross-platform support (see [here for more info](https://github.com/kaixxx/noScribe#development-and-contribution).
 
**Thank you, Hervé and all the great people here for the work you put into pyannote!**

**Comment by hbredin** [comment]
Looks great! Thanks for sharing.
